pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -62,16 +66,45 @@ func (f *Fit) Name() string {
 	return FitName
 }
 
+func validateFitArgs(args config.NodeResourcesFitArgs) error {
+	var allErrs field.ErrorList
+	resPath := field.NewPath("ignoredResources")
+	for i, res := range args.IgnoredResources {
+		path := resPath.Index(i)
+		if errs := metav1validation.ValidateLabelName(res, path); len(errs) != 0 {
+			allErrs = append(allErrs, errs...)
+		}
```

Comment:
does this tighten validation on an existing configuration file? what would have happened before if values that didn't pass ValidateLabelName were passed in this config field?

Reply:
It wouldn't have match any resources